rustfmt-core/rustfmt-lib/src/format_report_formatter.rs

```diff
@@ -49,16 +48,15 @@ pub struct FormatReportFormatter<'a> {
 impl<'a> Display for FormatReportFormatter<'a> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         let formatter = DisplayListFormatter::new(self.enable_colors, false);
-        let errors_by_file = &self.report.internal.borrow().0;
 
-        for (file, errors) in errors_by_file {
-            for error in errors {
+        for (file, errors) in self.report.format_result_as_rc().borrow().iter() {
+            for error in errors.errors_excluding_macro() {
```

Comment:
Why are macro errors excluded?

Reply:
Macro-related errors are filtered because they are for internal use, and we don't want users to see them.